website/docs/d/cloudant.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cloudant"
+description: |-
+  Get information about Cloudant instance.
+subcategory: "Cloud Databases"
+---
+
+# ibm_cloudant
+
+Provides a read-only data source for an existing IBM Cloud Cloudant service. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
Could add a reference link sentence like: For more information, about how to use this service, see[Cloudant documentation](https://cloud.ibm.com/docs/Cloudant?topic=Cloudant-getting-started-with-cloudant).